parse_logs.py

```diff
@@ -11,6 +11,11 @@
 
 requests = re.split("\[\d*\.\d*ms\]", data)
 result = [[time, request.strip()] for time, request in zip(time_ms, requests)]
+
+with open('unsort_'+file_name, 'w') as file_with_sorted_logs:
```

Comment:
name of file is incorrect, file have not sorted logs, but the variable has a name `sorted`
```suggestion
with open('unsort_'+file_name, 'w') as file_with_unsorted_logs:
```